test/support/dap_utils.rb

```diff
@@ -97,11 +97,11 @@ def connect_to_dap_server test_info
     end
 
     TIMEOUT_SEC = (ENV['RUBY_DEBUG_TIMEOUT_SEC'] || 10).to_i
-    dt = ENV['RUBY_DEBUG_PROTOCOL_TEST']
-    DAP_TEST = dt == 'true' || dt == '1'
+    pt = ENV['RUBY_DEBUG_PROTOCOL_TEST']
+    PROTOCOL_TEST = pt == 'true' || pt == '1'
```

Comment:
I think if we include `Protocol_TestUtils` before CDP/DAP utils, we can use the `PROTOCOL_TEST` defined in it?